config/locales/titles/en.yml

```diff
@@ -10,9 +10,10 @@ en:
       password: Edit your password
       phone: Edit your phone number
     enter_2fa_code: Enter the secure one-time security code
-    loa3_verified:
-      'true': You have verified your identity with %{app}
-      'false': You have created your account with %{app}
+    loa3_verified_html: You have %{accent} with %{app}
+    loa3_verified_accent:
```

Comment:
The completions page is only shown to users who have just completed creating their account. Once your account is already created/verified, signing in will never show the completions page. The user gets redirected to the SP immediately after 2FA.

If we want to show the attributes requested by the SP each time a user signs in, we'll have to update our logic in a separate PR.